types/redux-form/tsconfig.json

```diff
@@ -9,7 +9,7 @@
         "noImplicitAny": true,
         "noImplicitThis": true,
         "strictNullChecks": true,
-        "strictFunctionTypes": true,
+        "strictFunctionTypes": false,
```

Comment:
We generally don't like going back on the strictness provided by a library - what became the issue here, too?

Reply:
I should have linked to the issue I filed: https://github.com/DefinitelyTyped/DefinitelyTyped/issues/20414

The above issue is present in master, but was only revealed in this PR due to the fact that I touched the `redux-form` tests. I disabled this option to get the tests passing. I'm not familiar with that particular library, and making changes to it seemed out of scope here.

Reply:
Welp. I guess disabling it for now is OK, then.